media-gfx/librecad/librecad-2.0.7.ebuild

```diff
@@ -21,18 +21,15 @@ DEPEND="
 	dev-qt/qtsvg:4
 	dev-libs/boost
 	dev-cpp/muParser
-	media-libs/freetype
-	"
-RDEPEND="${DEPEND}"
+	media-libs/freetype"
 
-src_unpack() {
-	unpack ${A}
-	mv * ${P}
-}
+RDEPEND="${DEPEND}"
 
 src_prepare() {
 	# currently RS_VECTOR3D causes an internal compiler error on GCC-4.8
-	use 3d || sed -i -e '/RS_VECTOR2D/ s/^#//' librecad/src/src.pro
+	if ! use 3d; then
+		sed -i -e '/RS_VECTOR2D/ s/^#//' librecad/src/src.pro || die
+	fi
```

Comment:
Sure. I am always unsure about combined ORs, that why I try to use braces for readability ;)